registrations/migrations/0024_signup_and_signup_group_protected_data.py

```diff
@@ -0,0 +1,156 @@
+# Generated by Django 3.2.20 on 2023-09-22 12:26
+
+from django.db import migrations, models
+from django.db.models import deletion, Q
+import encrypted_fields.fields
+
+
+def forwards_func(apps, schema_editor):
+    SignUpGroup = apps.get_model("registrations", "SignUpGroup")  # noqa: N806
+    SignUpGroupProtectedData = apps.get_model(  # noqa: N806
+        "registrations", "SignUpGroupProtectedData"
+    )
+
+    signup_group_protected_datas = []
+    for signup_group in SignUpGroup.objects.filter(
+        Q(extra_info__isnull=False) & ~Q(extra_info__exact="")
+    ).only("id", "registration_id", "extra_info"):
+        signup_group_protected_datas.append(
+            SignUpGroupProtectedData(
+                registration=signup_group.registration_id,
+                signup_group=signup_group.id,
+                extra_info=signup_group.extra_info,
+            )
+        )
```

Comment:
I'm getting following error when trying to run migrations with `python manage.py migrate registrations`
```
Applying registrations.0024_signup_and_signup_group_protected_data...Traceback (most recent call last):
  File "/app/manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 89, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/commands/migrate.py", line 244, in handle
    post_migrate_state = executor.migrate(
  File "/usr/local/lib/python3.9/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python3.9/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python3.9/site-packages/django/db/migrations/executor.py", line 227, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/usr/local/lib/python3.9/site-packages/django/db/migrations/migration.py", line 126, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/usr/local/lib/python3.9/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/app/registrations/migrations/0024_signup_and_signup_group_protected_data.py", line 38, in forwards_func
    SignUpProtectedData(
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 485, in __init__
    _setattr(self, field.name, rel_obj)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/fields/related_descriptors.py", line 215, in __set__
    raise ValueError(
ValueError: Cannot assign "2": "SignUpProtectedData.registration" must be a "Registration" instance.
```

Should this be:
```
for signup_group in SignUpGroup.objects.filter(
        Q(extra_info__isnull=False) & ~Q(extra_info__exact="")
    ).only("id", "registration", "extra_info"):
        signup_group_protected_datas.append(
            SignUpGroupProtectedData(
                registration=signup_group.registration,
                signup_group=signup_group,
                extra_info=signup_group.extra_info,
            )
        )
```